samples/dash-if-reference-player/app/main.js

```diff
@@ -1537,6 +1550,8 @@ app.controller('DashController', ['$scope', '$window', 'sources', 'contributors'
             autoLoad: $scope.autoLoadSelected,
             muted: $scope.muted,
             drmToday: $scope.drmToday,
+            enableRollUp: $scope.imscEnableRollUp,
```

Comment:
We should not need these fields here as these two attributes are defined in `Settings.js`.

Reply:
Thanks, removed in 67a98bb.